config/routes.rb

```diff
@@ -206,6 +206,7 @@
   resources :tickets, only: [:index, :show] do
     post 'update_status' => 'tickets#update_status', as: :update_status
     get 'edit_person_validators' => 'tickets#edit_person_validators', as: :edit_person_validators
+    resources :comments, only: [:index, :create], module: :tickets
```

Comment:
What does `module: :tickets` do? I cannot find `module` being used anywhere else in our routes file (unless CTRL+F betrayed me) so I'm wondering whether it's necessary.